Geocoder/src/test/java/com/example/geocoder/services/CensusApiServiceTests.java

```diff
@@ -33,7 +33,7 @@ void submitAddressInvalidRequest() {
 
     AddressMatchResponse expectedAddressMatch = new AddressMatchResponse("Address Not Found");
     ResultResponse expectedResult = new ResultResponse(expectedAddressMatch);
-    CensusApiResponse expectedResponse = new CensusApiResponse(expectedResult);
+    CensusApiResponse expectedResponse = new CensusApiResponse(expectedResult, true);
```

Comment:
Question: I believe we can remove the newly rate limit bypass field from the CensusApiResponse as well as convert it back into a record. correct?